doc/cpp/probe_sample.rst

```diff
@@ -277,7 +276,7 @@ If the mechanism is not defined at a particular site, that site is ignored.
 
 *  Sample value: ``double``. State variable value.
 
-*  Metadata: ``mlocation``. Location as given in the probe address.
+*  Metadata: ``mlocation``. Location as given in the probeset address.
```

Comment:
Note to self: get rid of these 'm' prefixes.

Reply:
Shall I do it in this PR? Touches a few files.